code/modules/client/preference_setup/vore/07_traits.dm

```diff
@@ -255,6 +260,14 @@
 
 			var/conflict = FALSE
 
+			user.isSynthetic()	//Recheck just to be sure
```

Comment:
Is this meant to be `pref.dirty_synth = user.isSynthetic()`?

Reply:
🤔 That wasn't what i was trying to do, but it might work just as well if not better.

Reply:
Tested, didn't seem to quite work.

Reply:
Hrm, it just seems like this line doesn't actually do *anything* right now, I don't think isSynthetic() has any side effects?